aea/configurations/base.py

```diff
@@ -922,7 +923,7 @@ def __init__(
         )
         self.dependencies = dependencies if dependencies is not None else {}
         self.description = description
-        self.config = config
+        self.config = config if len(config) > 0 else {}
```

Comment:
when can it be length zero?

Reply:
when the keyword arguments are zero. See above the constructor `**config`